src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -35,7 +35,7 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
     private hasNativeNotebookCells: ContextKey;
     private isPythonFileActive: boolean = false;
     private isPythonNotebook: ContextKey;
-    private hasNativeNotebookOpen: ContextKey;
+    private hasNativeNotebookOrInteractiveWindowOpen: ContextKey;
```

Comment:
I thought about having NativeNotebookOpen and InteractiveWindowOpen and a merge for Notebook or InteractiveWindow. But that seemed like overkill as this was only being used for the variable view. So just lengthened the name to cover both cases.